### PR TITLE
#33 Fixes issues of detached tbody from table

### PR DESCRIPTION
**Pull request by sajus**
@j0k3r  can you please review the PR?

### CODE REVIEW

File: src/js/table.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ Table.prototype = {
         );
 
         var table = this._doc.getElementById('medium-editor-table');
+        var tbody = this._doc.getElementById('medium-editor-table-tbody');
+        if(!$(table).find('#medium-editor-table-tbody').length) {
```
Comment (j0k3r) [inline src/js/table.js]: Better do

``` js
0 === $(table).find('#medium-editor-table-tbody').length
```

**Review by j0k3r — COMMENTED** [review]
Any chance to add some tests about that?

File: src/js/table.js (hunk 1 above)
Comment (sajus) [reply src/js/table.js]: Okies that can be done!

**Comment by sajus** [comment]
This is an edge case where it doesn't occur all the time, I don't really think we need test cases here.

**Comment by sajus** [comment]
Also i will fix issues came under jscs task

**Comment by sajus** [comment]
@j0k3r , is this good now?

**Comment by j0k3r** [comment]
Looks good, thanks
